test/hotspot/jtreg/compiler/valhalla/inlinetypes/TestIntrinsics.java

```diff
@@ -1612,7 +1621,8 @@ public Test80Value2 test80(Test80Value1 v, boolean flat, long offset) {
     public void test80_verifier() throws Exception {
         Test80Value1 v = new Test80Value1();
         Field field = Test80Value1.class.getDeclaredField("v");
-        Asserts.assertEQ(test80(v, U.isFlatField(field), U.objectFieldOffset(field)), v.v);
+        var flat = U.isFlatField(field);
+        Asserts.assertEQ(test80(v, flat, flat ? U.fieldLayout(field) : 0, U.objectFieldOffset(field)), v.v);
```

Comment:
This is a good example showing why fieldLayout() should return a known value even if not flat.